Celeste.Mod.mm/Mod/Everest/Everest.cs

```diff
@@ -622,6 +619,15 @@ public static partial class Everest {
                 patch_Audio.System?.release();
             };
 
+            // Unix-likes can just fork-and-die to start the new game
+            if (Environment.OSVersion.Platform == PlatformID.Unix ||
+                Environment.OSVersion.Platform == PlatformID.MacOSX) {
+                Logger.Log(LogLevel.Info, "info", Path.Combine(PathGame, "Celeste"));
+                Process fork = new Process();
+                fork.StartInfo.FileName = Path.Combine(PathGame, "Celeste");
+                fork.Start();
```

Comment:
Any reason why the process is being forked in `_QuickFullRestart` and not on shutdown like before? Doing it too early might cause Steam to see the game as "still alive" and prevent it from running a second time.

Reply:
Yes. Shutdown is too late for Linux for whatever reason, and in all of my testing Steam waits plenty long enough for the parent Celeste process to die.